lib/ansible/modules/package_facts.py

```diff
@@ -259,12 +259,38 @@
 
 
 ALIASES = {
-    'rpm': ['dnf', 'dnf5', 'yum' , 'zypper'],
+    'dnf': ['dnf', 'dnf5'],
```

Comment:
This probably should not cover dnf5 as that uses a completely separate library which has a different API.

Are you interested in adding one for `dnf5` too?